src/transport_ofi.c

```diff
@@ -271,6 +311,181 @@ typedef enum{
 /* default CQ depth */
 uint64_t shmem_transport_ofi_max_poll = (1ULL<<30);
 
+
+/* STX control variables */
+enum stx_share_alg_t {
+    ROUNDROBIN = 0,
+    RANDOM
+};
+typedef enum stx_share_alg_t stx_share_alg_t;
+static stx_share_alg_t shmem_transport_ofi_stx_share_alg;
+static int stx_start_idx = 0;
+
+static long shmem_transport_ofi_stx_max;
+
+struct shmem_transport_ofi_stx_t {
+    struct fid_stx*   stx;
+    long              ref_cnt;
+    bool              is_private;
```

Comment:
I believe you need to ```#include <stdbool.h>``` for the ```bool``` type and ```true``` and ```false``` macros.

Reply:
🌮